core/src/main/java/hudson/scheduler/CronTabList.java

```diff
@@ -78,17 +96,31 @@ public static CronTabList create(String format) throws ANTLRException {
     public static CronTabList create(String format, Hash hash) throws ANTLRException {
         Vector<CronTab> r = new Vector<CronTab>();
         int lineNumber = 0;
+        String timezone = null;
+
         for (String line : format.split("\\r?\\n")) {
             lineNumber++;
             line = line.trim();
+            
+            if(lineNumber == 1 && line.startsWith("TZ=")) {
```

Comment:
> I understand that the idea is to set the timezone once and have it apply to all values, but I think the syntax would be more explicit if the TZ was defined on a line of its own

Actually, I see that I misread and this is *already* the case, i.e. the `TZ=` thing isn't a prefix for each line.  But the points are still valid.